media/js/views.js

```diff
@@ -41,13 +41,11 @@ $(document).ready( function() {
 
     // handles create buttons in dynamic/static interface view and range
     // detail view
-    $( document ).on( 'click', '#system_create', function( e ) {
-        e.preventDefault();
-        var csrfToken = $('#view-metadata').attr( 'data-csrfToken' );
-        button_to_form( this, csrfToken, function( postForm ){
-            $(postForm).submit();
-        });
-    });
+    //$( document ).on( 'click', '#system_create', function( e ) {
```

Comment:
Why is this commented out? If it's not doing anything I think it should just be removed.